### ISSUE TITLE
[CAIA A11y Research]: MHV Medications (May EOM & June 2024)

### ISSUE THREAD

**Issue by AnneCostello** [department-of-veterans-affairs/va.gov-team#82618]
> [!NOTE]
> **High-Level Overview**: `MHV Medications`
> - **Initial Kickoff**: `Date and Time TBD`
> - **Research Timeframe**: `EOM May 2024 and June 2024 (if needed)`
> - **Time of Day:** `Morning/Early Afternoon`
> - **CAIA AT Pilot**:  #83156 
> - **Assistive Tech Users**: `10` AT Users 
>.   - Source of Truth: [DSVA Slack Canvas](https://dsva.slack.com/docs/T03FECE8V/F07491XB411)
> - **Artifacts Available**: `5/8/2024`
> - **Testing Environment:** `Staging` _Prototypes available to review_
> - **Assistive Tech Being Used**: `Screen Readers (5)` `SR and Text-to-Speech (1)` `Magnifiers/Zoom` (5)
> - **Task Lists**: See task lists for next steps/definition of done

## CAIA Assignee(s)

<details><summary>Toggle to view information about the CAIA Team.</summary>

- `Sara`
- `SK`
- `Jamie`
- `Evan`

</details>

## About the Team

<details><summary>Toggle to view information about the team.</summary>

- Team name:Medications on VA.gov
- OCTO-DE product owner: Kay Lawyer
- Product manager: Kaitlin Fink 
- Designer: Lexi Wunder
- FE engineer:
- Product/team Slack channel: [#mhv-medications-rx](https://dsva.slack.com/archives/C04PRFEJQTY)

</details>

<details><summary>Toggle to view more intake research ticket information</summary>

### Do you have an accessibility champion?

No, we don't have a champion (yet)

### Calendar checking

- [X] Based on this calendar, confirm we have specialists who can support your study in your desired timeframe
- [X] Confirm you are filing this support ticket at least 1 week in advance of said timeframe, so we have enough heads up to provide support

### Have you reviewed the CAIA capacity calendar?

- [X] Yes, we have reviewed the CAIA capacity calendar and can confirm specialists are available to support in our research timeframe
</details>

## Research Sessions

**_May 28 - May 30, June 3 & 4 as alternates if needed_**

> [!TIP]
> - Providing CAIA A11ys with the time slots in advance for AT users, will allow a11ys to avoid conflicts on the CAIA A11y Capacity Calendar, and reach out to other a11ys, should CAIA not be available. 
> - Providing CAIA A11ys with the technology and devices the users will be testing with, will allow CAIA to prepare in advance for the sessions. 

<details><summary>Toggle to view research timing information.</summary>

### Timeframe

- **CAIA Pilot Session**: #83156 
- **Timeframe**: `5`/`28`/2024 - `6`/`4`/2024

### Length of sessions

- **Session length:** `90` minutes
- **Buffer time between sessions:** `30` minutes
- **Maximum sessions per day:** `pending`

</details>

###  Testing Schedule

CAIA A11Ys will be looking for device and AT information. We realize that some of this information will not be available until the day of the session. 

Please toggle and use the following table to enter the following information for each participant: Date and Time, Assistive tech (AT), Device and Operating System, Browser, and Skill level:

<details><summary>Toggle to view the AT testing schedule</summary>

P# | Date and Time (ET) | AT | Device and OS | Browser | Skill | CAIA A11ys Available
------------------|------------------|--------------|--------------|--------------|--------------|--------------
**PILOT** ✈ _with volunteer AT user_  | 5/20/24, 9:30-11:00 a.m. | Jaws | Desktop | Edge |  Advanced | Sara and Jamie
P3 | 5/23/24, 1-2 p.m.  | Magnifier | Desktop; Windows | Safari, Chrome, etc. `TBD` | Intermediate | Evan
P4 | 5/24/24, 9-10 a.m. | ~~Zoomtext~~ Magnifier | Mobile; iOS, iPhone | Chrome | Advanced | Jamie
P6 | 5/24/24, 1-2 p.m.| Zoom | Desktop, MacOS | Safari, Chrome, etc. `TBD` | Novice, Intermediate, Advanced `TBD` | Sara
P7 | 5/28/24, 9-10:30 a.m. | VoiceOver | iPad,  iOS | Safari, Chrome, etc. `TBD` | Novice, Intermediate, Advanced `TBD` | Sara
P9 | 5/29/24, 9-10:30 a.m. | VoiceOver and Voice Command | iPad, iOS | Safari | Intermediate | Jamie
P11 | 5/30/24, 9-10:30 a.m. | JAWS | Desktop/Laptop, Windows | Chrome | Advanced | Jamie
P12 | 5/30/24, 1-2:30 p.m. | JAWS | Desktop/Laptop, Windows | Safari, Chrome, etc. `TBD` | Novice, Intermediate, Advanced `TBD` | SK
⛔ P10 `rescheduled` `didn't use AT` | 6/4/24, 9-10:30 a.m. | ~~VoiceOver~~ TalkBack | ~~MacOS/Desktop/Laptop~~ Android, Mobile | `N/A` | Novice | Jamie
P15 | 6/6/24, 12-1 p.m. | Magnifier | Mobile or Desktop; Windows | Safari, Chrome, etc. `TBD` | Novice, Intermediate, Advanced `TBD` | SK
⛔ P16 `didn't use AT` | 6/6/24, 1:30-2:30 p.m. |~~ Magnifier~~ none | Desktop | Windows, Desktop | n/a | Jamie

- **Consideration**: You can also consider asking participants more specific AT and device combinations. 
- **For instance:** TalkBack on Samsung Galaxy 8, or VoiceOver on iPad with magnification.

</details>

**_CAIA A11ys will review the coverage needed, and self-assign, based on a11y availability._**

## Supporting artifacts 

_**Anne shared the prototype link so we can see what the environment will look like. MHV Medications will be using staging for the actual sessions.**_

<details><summary>Toggle to view the supporting artifacts</summary>

- [Prototype or mockups](https://www.figma.com/proto/UGHasFKuKENvKz8AgSmxtI/Medications?node-id=4942-23559&t=5AGkqQpTPeDlSNuA-1&scaling=scale-down&page-id=4942%3A23558&starting-point-node-id=4942%3A23559)
- [Research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/mhv-to-va.gov/medications/research/2024-05-medications-usability-testing-round3-AT/research-plan.md)
- [Research conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/mhv-to-va.gov/medications/research/2024-05-medications-usability-testing-round3-AT/conversation-guide-sr-vc.md)

</details>

## Next Steps: Definition of Done

```[tasklist]
### CAIA A11Y Tasks
- [x] **INTAKE TICKET:** @sara-amanda to process intake ticket
- [ ] **INITIAL MEETING:** Schedule meeting with team
- [ ] **(OPTIONAL) FEEDBACK REQUESTED:** Review Conversation Guide
- [x] **CAIA PILOT SESSION: SCHEDULE** Pilot session with team.
- [x] **CAIA PILOT SESSION: CREATE TICKET** `#83156`
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/83156
- [x] **CALENDAR HOLDS** Update A11y [CAIA Capacity Calendar](https://calendar.google.com/calendar/u/0/embed?src=c_f982350fc73a8162a1088ab14d00dd37335b2b559a60d6b67eb8021416f81c2a@group.calendar.google.com&ctz=America/Chicago) with holds for user research sessions with AT, when dates become available.
- [x] **REMOVE CALENDAR HOLDS** Remove holds on A11y [**CAIA Capacity Calendar**](https://calendar.google.com/calendar/u/0/embed?src=c_f982350fc73a8162a1088ab14d00dd37335b2b559a60d6b67eb8021416f81c2a@group.calendar.google.com&ctz=America/Chicago) and replace with invitations from project team (or Perigean)
- [x] Updated Calendar Holds with Additional Sessions 5/22/2024
- [x] **RESEARCH NOTES** Create [CAIA notes document](https://docs.google.com/document/d/1iJ_2rwra9Qj6rv0JZzSBs80vwYlyyGZZDpBifdzbQhw/edit?usp=sharing)
- [x] **Divvy user research session support with CAIA A11ys** - based on availability and technology available to support.
- [x] **Review research folder and assets**
- [x] **Participate in research** **Date Span:**`5/28/2024` and `6/6/2024`
- [x] **SESSION NOTES**: Use [CAIA notes document](https://docs.google.com/document/d/1iJ_2rwra9Qj6rv0JZzSBs80vwYlyyGZZDpBifdzbQhw/edit?usp=sharing) during the sessions
- [x] **SYNC:** Connect with CAIA A11ys on research findings (meeting)
- [ ] **Create/update the CAIA A11y Research Deliverable** This is dependent on the number of study participants CAIA supports. If minimal, A11Ys may share feedback only in the feedback Mural or this ticket.
- [ ] **Share the completed deliverable** with the project team via Slack: `Primary Team Slack` and `Ticket`
```

```[tasklist]
### VFS TEAM Tasks
- [x] Tag @coforma-terry in Slack [#sitewide-content-accessibility-ia](https://dsva.slack.com/archives/C01K37HRUAH)
- [x] Schedule intial meeting with @coforma-terry and @sara-amanda `Slacked`
- [x] Schedule a CAIA AT pilot session with CAIA a11ys `#83156`
- [ ] (OPTIONAL) Receive feedback on Conversation Guide for Screen Readers (if requested)
- [x] **PROTOTYPE:** Provide prototype to CAIA A11ys via this ticket in the supporting artifacts section
- [x] **DATES NEEDED** See `Table` in the ticket above. Please provide date range and slots for AT sessions being planned.
- [x] **ASSISTIVE TECH. INFO NEEDED:** Please provide the AT the users will be utilizing during the research sessions in `Table` in the ticket above.
- [ ] AT Info: AT Browser, skill level, years of use for AT to be asked during sessions
- [ ] **INVITES:** Forward invites from Perigean to CAIA A11ys (Sara or Terry)
```

**Comment by sara-amanda** [comment]
## Pilot Scheduling

@AnneCostello here is the pilot ticket: #83156 - if you can let us know, [Slack is good](https://dsva.slack.com/archives/C8E985R32/p1715786583306349?thread_ts=1715179764.034099&cid=C8E985R32), or here, or both, what dates/times would work for a pilot with AT and a CAIA Team Member. 😄 

cc: @coforma-terry @SarahKay8 @EvanAtCoforma  @coforma-jamie

**Comment by sara-amanda** [comment]
## AT Sessions 
- [x] Added AT sessions to table in this ticket, from DSVA MHV Medications Canvas (Source of Truth)
- [x] Added CAIA A11Y names to the canvas.
- [x] Updated old P16 to P15*
- [x] Updating Research Tracker
- [x] Updating Notes Document

### Prepping for Sessions: AT Info.
Additional information needed, when available @AnneCostello , if you are able to obtain it from Perigean in advance, to help us prep for the sessions, and/or in the actual session, that would be great! 
- [ ] Device (i.e. Mobile, desktop, tablet)
- [ ] Browser (i.e. Chrome, Safari, Firefox, Edge, etc.)
- [ ] Skill Level - self-assessed (i.e. Advanced, Intermediate or Novice)
- [ ] Length of time using AT (duration in years, or months)

cc: @SarahKay8 @coforma-jamie @EvanAtCoforma @coforma-terry

**Comment by coforma-jamie** [comment]
All findings were shared with the team at the end of each research session via Slack.
